### DISCUSSION TITLE
Issue Management

### DISCUSSION THREAD

**Discussion by PaulWoitaschek** [PaulWoitaschek/Voice#1330] [Announcements]
Hello Everyone!

In order to clean up the huge issue board and to not lose valuable suggestions I decided to use the Github discussions for general ideas, feature upvotes and questions.
So when you have an idea on what would be great to have in voice, use the:

👉 [Ideas 💡 ](https://github.com/PaulWoitaschek/Voice/discussions/categories/ideas) 👈 

Please upvotes ideas that you like. This helps with prioritizing and determining, how many users would benefit from a feature.

If you have a general question, use the

👉  [Q&A ❓ Section](https://github.com/PaulWoitaschek/Voice/discussions/categories/q-a) 👈

**Comment by kingf201777** [comment]
feat(alipay): 淘宝小程序支持ar-camera组件

**Comment by luzpaz** [comment]
There are certain duplicate ideas in the ideas discussions.
